### ISSUE TITLE
Pods breaks TinyMCE text editing in WP 5.0 (maybe also related to Yoast SEO)

### ISSUE THREAD

**Issue by jeiseman** [pods-framework/pods#5229]
After upgrading to WordPress 5.0 I found that clicking on the Text tab in TinyMCE (for editing a post) shows a blank page. I am editing with the Classic Editor. In the developer console I see the following error:
`Uncaught TypeError: Cannot read property 'isSavingPost' of undefined
    at pods-dfv.js:104
    at data.min.js:1
    at Array.forEach (<anonymous>)
    at n (data.min.js:1)
    at data.min.js:1
    at b (data.min.js:1)
    at data.min.js:1
    at Object.dispatch (data.min.js:1)
    at e.value (wp-seo-post-scraper-921.min.js:1)
    at e.value (wp-seo-post-scraper-921.min.js:1)`

Note that I also have Yoast SEO installed.

## Pods Version
2.7.10

## WordPress Environment
WordPress 5.0. Google Chrome. Windows 10.
```
WordPress Version: 5.0

PHP Version: 7.2.12

MySQL Version: 5.6.40

Server Software: Apache

Your User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36

Session Save Path: /tmp

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: No
Memcached: Yes
Redis: No
WPDB Prefix: wp_

WP Multisite Mode: No

WP Memory Limit: 40M

Current Memory Usage: 13.648M

Current Memory Usage (real): 16.000M

Pods Network-Wide Activated: No

Pods Install Location: /home/class211/staging/2/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: GeneratePress Child

Currently Active Plugins:

Accelerated Mobile Pages: 0.9.97.22
AdRotator: 2.0.3
Advanced Image Styles: 0.4.1
Akismet Anti-Spam: 4.1
All In One WP Security: 4.3.7.2
Cache-Control: 2.2.2
Classic Editor: 1.1
Debug This: 0.6.1
Duplicator: 1.2.52
GigPress: 2.3.22
Google Analytics for WordPress by MonsterInsights: 7.3.2
GP Premium: 1.7.5
Heartbeat Control: 1.2.5
iframe: 4.3
Jetpack by WordPress.com: 6.8.1
mobble: 1.6
Open Graph for Facebook, Google+ and Twitter Card Tags: 2.2.4.2
Optimize Database after Deleting Revisions: 4.7.2
Pods - Custom Content Types and Fields: 2.7.10
Rich Text Excerpts: 1.3.4
SG Optimizer: 5.0.7
Tabify Edit Screen: 0.9.7
TinyMCE Advanced: 4.8.0
User Role Editor: 4.47
Widget Importer & Exporter: 1.5.3
WP Crontrol: 1.6.2
WP Show Posts: 1.1.3
WP Show Posts Pro: 0.5
WP to Twitter: 3.3.9
XML Sitemap & Google News: 5.0.7
Yoast SEO: 9.2.1
```
</details>

## Pods Package Export (helpful!)
```
{"meta":{"version":"2.7.10","build":1544189878},"pods":{"37655":{"id":37655,"name":"user","label":"User","description":"","type":"user","storage":"meta","object":"user","alias":"","fields":{"short-bio":{"id":37656,"name":"short-bio","label":"Short Bio","description":"Short biography to be added below posts","help":"","class":"","type":"wysiwyg","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"0","wysiwyg_editor":"tinymce","wysiwyg_media_buttons":"1","wysiwyg_oembed":"0","wysiwyg_wptexturize":"1","wysiwyg_convert_chars":"1","wysiwyg_wpautop":"1","wysiwyg_allow_shortcode":"0","pick_allow_html":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","wysiwyg_repeatable":"0","wysiwyg_allowed_html_tags":""}},"show_in_menu":1,"rest_enable":"0","rest_base":"user","read_all":"0","write_all":"0"}}}
```

**Workaround**
Disable Pods

**Comment by jimtrue** [comment]
@jeiseman This sounds like this error #5225 which was reported yesterday and we have a patch for this one already. We probably will be pushing this one out as a maintenance release today.

**Comment by jimtrue** [comment]
I'm closing this as a Duplicate of #5225 which has been patched in #5226 I've already alerted @pglewis and @sc0ttkclark that we need to escalate pushing this fix out live.
